### PR TITLE
chore(ci): allow gpu workflows to run fast deduplicated default benchmarks

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: .github/workflows/integer_multi_bit_gpu_benchmark.yml [hunk 1]
```diff
@@ -4,10 +4,15 @@ name: Integer GPU Multi-bit benchmarks
 on:
   workflow_dispatch:
     inputs:
-      full_benchmark:
+      all_precisions:
         description: "Run all precisions"
         type: boolean
         default: false
+      # Run only deduplicated default operations with no scalar variants
```
Comment (IceTDrinker) [inline .github/workflows/integer_multi_bit_gpu_benchmark.yml:11]: can be put in the comment about the check box

File: .github/workflows/integer_multi_bit_multi_gpu_benchmark.yml [hunk 2]
```diff
@@ -4,10 +4,15 @@ name: Integer multi GPU Multi-bit benchmarks
 on:
   workflow_dispatch:
     inputs:
-      full_benchmark:
+      all_precisions:
         description: "Run all precisions"
         type: boolean
         default: false
+      # Run only deduplicated default operations with no scalar variants
```
Comment (IceTDrinker) [inline .github/workflows/integer_multi_bit_multi_gpu_benchmark.yml:11]: same thing can be put in the description

File: .github/workflows/integer_multi_bit_multi_gpu_benchmark.yml [hunk 3]
```diff
@@ -24,6 +29,7 @@ env:
   SLACK_USERNAME: ${{ secrets.BOT_USERNAME }}
   SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
   FAST_BENCH: TRUE
```
Comment (IceTDrinker) [inline .github/workflows/integer_multi_bit_multi_gpu_benchmark.yml:31]: is FAST_BENCH still used in the benchmarks code ?

**Review by IceTDrinker — COMMENTED** [review]
Small remark and a question about whether FAST_BENCH is still used in the bench code

File: .github/workflows/integer_multi_bit_multi_gpu_benchmark.yml (hunk 3 above)
Comment (soonum) [reply .github/workflows/integer_multi_bit_multi_gpu_benchmark.yml:31]: Yes, it deals with the precision selection

**Review by IceTDrinker — APPROVED** [review]

**Pull request merged**